lib/h3.rb

```diff
@@ -1,11 +1,14 @@
-require "h3/h3"
-require "h3/version"
+require "h3/structs"
+
 require 'ffi'
 
 module H3
   extend FFI::Library
   ffi_lib ["libh3", "libh3.1"]
   H3_INDEX = :ulong_long
+  H3_TO_STR_BUF_SIZE = 32
+  H3_STR = FFI::MemoryPointer.new(:char, H3_TO_STR_BUF_SIZE)
```

Comment:
Not to kibitz too much, but if you put the memory allocation out here as a constant in the bindings, then the bindings will not be threadsafe and you won't be able to use `parallel` or other libraries with it, since multiple binding methods might be using the string scratch space to put H3Indexes into.

Reply:
Is that going to allocate the buffer in the heap in every call? Or is there a way to have the equivalent of a local `char[H3_TO_STR_BUF_SIZE]`?

Reply:
It'll allocate in the heap. We can call `free` on it, though - FFI provides a way to do it.

Perhaps I should go back through the code and ensure we're freeing heap memory.